libraries/Microsoft.Bot.Builder.Dialogs/DialogContainer.cs

```diff
@@ -12,7 +12,7 @@ protected DialogContainer(string dialogId = null)
         {
         }
 
-        protected DialogSet Dialogs { get; set; } = new DialogSet();
+        public DialogSet Dialogs { get; set; } = new DialogSet();
```

Comment:
>public [](start = 8, length = 6)

AdaptiveDialog had public property which is an fxcop error, 

This should be public